gdcdictionary/schemas/gene_drug_association.yaml

```diff
@@ -0,0 +1,76 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "protein_drug_association"
+title: ProteinDrugAssociation
+type: object
+namespace: http://gdc.nci.nih.gov
+program: '*'
+project: '*'
+category: reference
+description: Definitions for protein-Drug Associations
+additionalProperties: false
+submittable: true
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - created_datetime
+  - updated_datetime
+
+required:
+  - submitter_id
+  - project_id
+
+uniqueKeys:
+  - [id]
+  - [project_id, submitter_id]
+
+links:
+  - backref: protein_drug_association
+    label: proteins
+    multiplicity: many_to_many
```

Comment:
If this is `many_to_many`, then `proteins` needs to be an array

Reply:
`one_to_one` in commit b4a6400